packages/actor-init-sparql/config/config-default.json

```diff
@@ -87,6 +94,11 @@
             "@type": "MediatorCombineUnion",
             "cc:Mediator/bus": { "@id": "cbss:Bus/SparqlSerialize" },
             "cmcu:Mediator/CombineUnion/field": "mediaTypes"
+          },
+          "mediatorContextPreprocess": {
```

Comment:
Is this not supposed to be assigned to an actor somewhere? Or are there some defaults I'm missing that do this?

Reply:
Yes, this is the SPARQL init actor.